src/users/handlers/list.ts

```diff
@@ -0,0 +1,33 @@
+import { DynamoDB } from "aws-sdk";
+import { Context, APIGatewayProxyCallback, APIGatewayEvent } from "aws-lambda";
+import { AppEnv } from "@helpers/env";
+const dynamoClient = new DynamoDB.DocumentClient();
+
+export const handle = async (
+    event: APIGatewayEvent,
+    context: Context,
+    callback: APIGatewayProxyCallback
+): Promise<void> => {
+    console.log("User list");
+    const params = {
+        TableName: AppEnv.dynamoDbTableName,
+        Key: {},
+    };
+
+    dynamoClient.get(params, (error, result) => {
```

Comment:
@jerlin-simelabs `get` will always return only one item, you will have to use scan with `Limit` option here. https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#scan-property

Reply:
Need to move this logic  to the repository file. Check `create.ts`, as the goto reference.